src/languages/en.ts

```diff
@@ -2174,6 +2174,8 @@ export default {
             existingCategoryError: 'A category with this name already exists.',
             invalidCategoryName: 'Invalid category name.',
             importedFromAccountingSoftware: 'The categories below are imported from your',
+            glCode: 'GL code',
```

Comment:
I thinkkkkkkk we often want to have capital C, but could be good to confirm that with someone other than me 😅 

```suggestion
            glCode: 'GL Code',
```

Reply:
it's small c in the doc, so i'll stick to that. We follow sentence case

Reply:
minnnnnteresting, then ya i'd go with what's in the HL of the doc 👍